source/index.html.md

```diff
@@ -290,18 +290,9 @@ Choose archetype:
 1: https://snaplogiceng.jfrog.io/artifactory/thirdparty/ -> com.snaplogic.tools:SnapArchetype (An archetype that creates a Snap Pack, with example Snaps provided)
 Choose a number or apply filter (format: [groupId:]artifactId, case sensitive contains): : 1
 Choose com.snaplogic.tools:SnapArchetype version: 
-1: 4.25
-2: 4.26
-3: 4.27
-4: 4.28
-5: 4.29
-6: 4.30
-7: 4.31
-8: 4.32
-9: 33.0
-10: 34.0
-11: 35.0
-Choose a number: 11:
+1: 35.0
+2: 36.0
+Choose a number: 2:
```

Comment:
To keep people from using old versions, I've cut down the versions in our archetype catalog, but the older versions are still deployed.

Reply:
Looks good, I'd suggest putting the newest version on top.
```suggestion
1: 36.0
2: 35.0
Choose a number: 1:
```

Reply:
we can't change the order, archetype always shows (and defaults) to the latest version, the reason this is `Choose a number: 2:` is because it automatically selects 2 if you hit enter.  This is the direct output from the archetype:generate command.